Remotion/ObjectBinding/Web.Test/IndividualControlTests/BocEnumValueUserControl.ascx.cs

```diff
@@ -106,8 +106,10 @@ override protected void OnLoad (EventArgs e)
 
     GenderField.LoadUnboundValue ((Gender?)null, IsPostBack);
 
+    UnboundMarriageStatusField.Property = (IBusinessObjectEnumerationProperty) CurrentObject.BusinessObjectClass.GetPropertyDefinition("MarriageStatus");
     //UnboundMarriageStatusField.LoadUnboundValue (person.MarriageStatus, IsPostBack);
     UnboundReadOnlyMarriageStatusField.LoadUnboundValue (person.MarriageStatus, IsPostBack);
+    DisabledUnboundMarriageStatusField.Property = (IBusinessObjectEnumerationProperty) CurrentObject.BusinessObjectClass.GetPropertyDefinition("MarriageStatus");
```

Comment:
Mir kommen die changes unrelated vor?

Reply:
That's why the commit doesn't have a task-number to it. The change itself is needed at this time to fix a problem on the testpage impacting this issue. If it where production code, this would be a bug and a bugfix-branch. Since it's just test-code, it's fine.